app/src/main/java/org/p2p/wallet/history/db/dao/TransactionDao.kt

```diff
@@ -17,9 +17,9 @@ import org.p2p.wallet.history.db.entities.UnknownTransactionEntity
 @Dao
 abstract class TransactionDao<TRANSACTION : TransactionEntity>(private val tableName: String) {
     @RawQuery
-    abstract suspend fun getTransactionsByQuery(query: SupportSQLiteQuery): List<TRANSACTION>
+    protected abstract fun getTransactionsByQuery(query: SupportSQLiteQuery): List<TRANSACTION>
 
-    suspend fun getTransactionsBySignature(signatures: List<String>): List<TRANSACTION> {
```

Comment:
COROUTINES EXPERTS, ATTENTION PLEASE

i removed suspend here, now Repository guarantees that dao methods work on IO thread. Whatcha think?